packages/osd-plugin-helpers/src/cli.ts

```diff
@@ -94,17 +94,12 @@ export function runCli() {
           plugin
         );
 
-        if (semver.satisfies(opensearchDashboardsVersion, '<7.9')) {
+        if (semver.satisfies(opensearchDashboardsVersion, '>=2.0.0')) {
```

Comment:
For this, we will need to make main to `2.0` right ?

Reply:
This check is good for 1.x and current main. 
If later we have 2.x and main is 2.x, then we need to update here.

Reply:
this condition means, dashboard version should be `>= 2.0.0` is that correct ? our current version is 1.0

Reply:
Actually we can remove these two condition completely, we are mostly on 7.10 which the issue doesn't persist.  @ananzh  let's remove it.

Reply:
Will do. Thanks for providing me the issue link.